spec/integration/home_controller_spec.rb

```diff
@@ -0,0 +1,23 @@
+require_relative '../spec_helper'
+
+feature 'my profile page' do
```

Comment:
feel free to add it to [the other exceptions](https://github.com/foodcoops/foodsoft/blob/427561e5ae9bc07585aa727e7d73bd8ebe2315ad/.rubocop_todo.yml#L403-L413) for now.